tendrl/node_agent/manager/pull_hardware_inventory.py

```diff
@@ -122,6 +122,98 @@ def getTendrlContext():
     return tendrl_context
 
 
+def getNodeDisk():
+    '''returns structure
+
+    {"nodename": [{"DevName":   "devicename",
+                  "FSType":     "fstype",
+                  "FSUUID":     "uuid",
+                  "Model":      "model",
+                  "MountPoint": ["mountpoint", ...],
+                  "Name":       "name",
+                  "Parent":     "parentdevicename",
+                  "Size":       uint64,
+                  "Type":       "type",
+                  "Used":       boolean,
+                  "SSD":        boolean,
+                  "Vendor":     "string",
+                  "DiskId":     "uuid"}, ...], ...}
+    '''
+    devlist = {}
+    dev_info = {}
+    rv = []
+    columes = 'NAME,KNAME,FSTYPE,MOUNTPOINT,UUID,PARTUUID,MODEL,SIZE,TYPE,' \
+              'PKNAME,VENDOR'
+    keys = columes.split(',')
+    lsblk = ("lsblk --all --bytes --noheadings --output='%s' --path --raw" %
```

Comment:
This should mean an explicit dependency on the util-linux package in the rpm spec file. If this isn't already the case, please add this dependency to the spec, increase the release version and commit the spec file along with this pull request too.